src/tests/functional.spec.jsx

```diff
@@ -110,4 +110,22 @@ describe('functional tests', () => {
     assertSheet(sheet)
     wrapper.unmount()
   })
+
+  it('should update all dynamic props', () => {
```

Comment:
I think it would be better to name this test like 'should update dynamic props for media-rules'